st2common/st2common/exceptions/db.py

```diff
@@ -16,6 +16,10 @@
 from st2common.exceptions import StackStormBaseException
 
 
+class StackStormDBKeyNotFoundError(StackStormBaseException):
```

Comment:
We can remove the definition of StackStormDBKeyNotFoundError in the st2common.exceptions.db package as we've now got it in st2common.exceptions.keyvalue.DataStoreKeyNotFoundError exception instead. So you should be able to remove this definition.